scripts/migration.sh

```diff
@@ -34,6 +34,12 @@ printf "Generating migration boilerplate for patch level $NEW_LEVEL..."
 
 echo "SET NAMES utf8mb4 COLLATE utf8mb4_bin;\n" > "$FWD_SCHEMA"
 echo "-- TODO: Implement your forward migration here\n" >> "$FWD_SCHEMA"
+
+echo "SELECT value INTO @patchlevel FROM `dbMetadata` WHERE `name` = `schema-patch-level`;"
+echo "IF @patchlevel != $PREV_LEVEL THEN
+    SIGNAL SQLSTATE '45000' SET MYSQL_ERRNO = 1643, MESSAGE_TEXT = 'Could not migrate to next patch';
+  END IF;"
```

Comment:
Let's use the same convention the rest of this script follows, and keep it to one line of text per `echo` command. If we want to embed any extra newlines for padding in the output, include a `\n` at the end of the echo-text.